corev_apu/bootrom/gen_dts.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
```

Comment:
Thanks a lot for this! Could you add a license header?

Reply:
Yes. Done.